docs/painless/painless-api-reference/painless-api-reference-score/index.asciidoc

```diff
@@ -0,0 +1,472 @@
+* <<painless-api-reference-score-org-elasticsearch-index-fielddata-ScriptDocValues.GeoPoints, ScriptDocValues.GeoPoints>>
+* <<painless-api-reference-score-org-elasticsearch-index-fielddata-ScriptDocValues.Longs, ScriptDocValues.Longs>>
+* <<painless-api-reference-score-org-elasticsearch-index-fielddata-ScriptDocValues.Strings, ScriptDocValues.Strings>>
+
+
+==== org.elasticsearch.index.mapper
+<<painless-api-reference-score-org-elasticsearch-index-mapper, Expand org.elasticsearch.index.mapper>>
+
+* <<painless-api-reference-score-org-elasticsearch-index-mapper-IpFieldMapper.IpFieldType.IpScriptDocValues, IpFieldMapper.IpFieldType.IpScriptDocValues>>
+
+
+==== org.elasticsearch.index.query
+<<painless-api-reference-score-org-elasticsearch-index-query, Expand org.elasticsearch.index.query>>
+
+* <<painless-api-reference-score-org-elasticsearch-index-query-IntervalFilterScript.Interval, IntervalFilterScript.Interval>>
+* <<painless-api-reference-score-org-elasticsearch-index-query-VectorScriptDocValues, VectorScriptDocValues>>
+* <<painless-api-reference-score-org-elasticsearch-index-query-VectorScriptDocValues.DenseVectorScriptDocValues, VectorScriptDocValues.DenseVectorScriptDocValues>>
+* <<painless-api-reference-score-org-elasticsearch-index-query-VectorScriptDocValues.SparseVectorScriptDocValues, VectorScriptDocValues.SparseVectorScriptDocValues>>
+
+
+==== org.elasticsearch.index.similarity
+<<painless-api-reference-score-org-elasticsearch-index-similarity, Expand org.elasticsearch.index.similarity>>
+
+* <<painless-api-reference-score-org-elasticsearch-index-similarity-ScriptedSimilarity.Doc, ScriptedSimilarity.Doc>>
+* <<painless-api-reference-score-org-elasticsearch-index-similarity-ScriptedSimilarity.Field, ScriptedSimilarity.Field>>
+* <<painless-api-reference-score-org-elasticsearch-index-similarity-ScriptedSimilarity.Query, ScriptedSimilarity.Query>>
+* <<painless-api-reference-score-org-elasticsearch-index-similarity-ScriptedSimilarity.Term, ScriptedSimilarity.Term>>
+
+
+==== org.elasticsearch.painless.api
+<<painless-api-reference-score-org-elasticsearch-painless-api, Expand org.elasticsearch.painless.api>>
+
+* <<painless-api-reference-score-org-elasticsearch-painless-api-Debug, Debug>>
+
+
+==== org.elasticsearch.script
+<<painless-api-reference-score-org-elasticsearch-script, Expand org.elasticsearch.script>>
+
+* <<painless-api-reference-score-org-elasticsearch-script-JodaCompatibleZonedDateTime, JodaCompatibleZonedDateTime>>
+* <<painless-api-reference-score-org-elasticsearch-script-org.elasticsearch.script.ScoreScript, org.elasticsearch.script.ScoreScript>>
+
+
+==== org.elasticsearch.search.lookup
+<<painless-api-reference-score-org-elasticsearch-search-lookup, Expand org.elasticsearch.search.lookup>>
+
+* <<painless-api-reference-score-org-elasticsearch-search-lookup-FieldLookup, FieldLookup>>
+
+
+include::painless-api-reference-score-java-lang.asciidoc[]
+include::painless-api-reference-score-java-math.asciidoc[]
```

Comment:
I can't find this file in the PR. Is github's UI eating it?

Reply:
They're added now :).  Forgot to add the new files here.